### DISCUSSION TITLE
e8820s mt7621 编译报错，一直卡在这里

### DISCUSSION THREAD

**Discussion by hhhf1231** [coolsnowwolf/lede#12470] [Q&A / 快速问答]
-gcc-ranlib" STRIP=mipsel-openwrt-linux-musl-strip OBJCOPY=mipsel-openwrt-linux-musl-objcopy OBJDUMP=mipsel-openwrt-linux-musl-objdump SIZE=mipsel-openwrt-linux-musl-size CROSS="mipsel-openwrt-linux-musl-" ARCH="mipsel" clean compile;
make[4]: Entering directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
rm -f po2lmo parser.so version.lua *.o
mipsel-openwrt-linux-musl-gcc  -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include  -DPIC -fpic -c -o template_parser.o template_parser.c
mipsel-openwrt-linux-musl-gcc  -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include  -DPIC -fpic -c -o template_utils.o template_utils.c
mipsel-openwrt-linux-musl-gcc  -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include  -DPIC -fpic -c -o template_lmo.o template_lmo.c
mipsel-openwrt-linux-musl-gcc  -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include  -DPIC -fpic -c -o template_lualib.o template_lualib.c
mipsel-openwrt-linux-musl-gcc -L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/lib -L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/lib -znow -zrelro  -shared -o parser.so template_parser.o template_utils.o template_lmo.o template_lualib.o
./mkversion.sh version.lua git-24.182.36151-66d29de "LuCI Master"
make[4]: Leaving directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
CFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " CXXFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -mips16 -minterlink-mips16  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " LDFLAGS="-L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/lib -L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/lib -znow -zrelro " FPIC="-DPIC -fpic" LUCI_VERSION="git-24.182.36151-66d29de" LUCI_GITBRANCH="LuCI Master" make -C /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src/ AR="mipsel-openwrt-linux-musl-gcc-ar" AS="mipsel-openwrt-linux-musl-gcc -c -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro" LD=mipsel-openwrt-linux-musl-ld NM="mipsel-openwrt-linux-musl-gcc-nm" CC="mipsel-openwrt-linux-musl-gcc" GCC="mipsel-openwrt-linux-musl-gcc" CXX="mipsel-openwrt-linux-musl-g++" RANLIB="mipsel-openwrt-linux-musl-gcc-ranlib" STRIP=mipsel-openwrt-linux-musl-strip OBJCOPY=mipsel-openwrt-linux-musl-objcopy OBJDUMP=mipsel-openwrt-linux-musl-objdump SIZE=mipsel-openwrt-linux-musl-size CROSS="mipsel-openwrt-linux-musl-" ARCH="mipsel" DESTDIR="/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install"  install;
make[4]: Entering directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
mkdir -p /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/template
cp parser.so /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/template/parser.so
cp version.lua /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/version.lua
make[4]: Leaving directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
touch /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.built
rm -rf /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base.installed /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base
mkdir -p /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/luasrc ]; then install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/usr/lib/lua/luci; cp -pR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/luasrc/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/usr/lib/lua/luci/; find /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/usr/lib/lua/luci/ -type f -name '*.luadoc' | xargs -r rm; true; 	find /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/usr/lib/lua/luci/ -type f -name '*.htm' | while read src; do /home/xiao/lede/staging_dir/host/bin/sed -i -e 's/<%# *\([^ ]*\)PKG_VERSION *%>/\1git-24.182.36151-66d29de/g' -e 's/"\(<%= *\(media\|resource\) *%>[^"]*\.\(js\|css\)\)"/"\1?v=git-24.182.36151-66d29de"/g' "$src"; done; else true; fi
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/htdocs ]; then install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/www; cp -pR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/htdocs/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/www/; else true; fi
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/root ]; then install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/; cp -pR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/root/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/; else true; fi
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src ]; then CFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " CXXFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -mips16 -minterlink-mips16  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " LDFLAGS="-L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/lib -L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/lib -znow -zrelro " FPIC="-DPIC -fpic" LUCI_VERSION="git-24.182.36151-66d29de" LUCI_GITBRANCH="LuCI Master" make -C /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src/ AR="mipsel-openwrt-linux-musl-gcc-ar" AS="mipsel-openwrt-linux-musl-gcc -c -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro" LD=mipsel-openwrt-linux-musl-ld NM="mipsel-openwrt-linux-musl-gcc-nm" CC="mipsel-openwrt-linux-musl-gcc" GCC="mipsel-openwrt-linux-musl-gcc" CXX="mipsel-openwrt-linux-musl-g++" RANLIB="mipsel-openwrt-linux-musl-gcc-ranlib" STRIP=mipsel-openwrt-linux-musl-strip OBJCOPY=mipsel-openwrt-linux-musl-objcopy OBJDUMP=mipsel-openwrt-linux-musl-objdump SIZE=mipsel-openwrt-linux-musl-size CROSS="mipsel-openwrt-linux-musl-" ARCH="mipsel" DESTDIR="/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install"  install; cp -fpR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/; else true; fi
make[4]: Entering directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
mkdir -p /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/template
cp parser.so /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/template/parser.so
cp version.lua /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/version.lua
make[4]: Leaving directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
touch /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base.installed
mkdir -p /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/root-ramips/stamp
SHELL= flock /home/xiao/lede/tmp/.root-copy.flock -c 'cp -fpR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-base/. /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/root-ramips/'
touch /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/root-ramips/stamp/.luci-base_installed
removed '/home/xiao/lede/bin/packages/mipsel_24kc/luci/luci-base_git-24.182.36151-66d29de-1_mipsel_24kc.ipk'
mkdir -p /home/xiao/lede/bin/targets/ramips/mt7621/packages /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/CONTROL /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/pkginfo
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/luasrc ]; then install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/usr/lib/lua/luci; cp -pR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/luasrc/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/usr/lib/lua/luci/; find /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/usr/lib/lua/luci/ -type f -name '*.luadoc' | xargs -r rm; true; 	find /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/usr/lib/lua/luci/ -type f -name '*.htm' | while read src; do /home/xiao/lede/staging_dir/host/bin/sed -i -e 's/<%# *\([^ ]*\)PKG_VERSION *%>/\1git-24.182.36151-66d29de/g' -e 's/"\(<%= *\(media\|resource\) *%>[^"]*\.\(js\|css\)\)"/"\1?v=git-24.182.36151-66d29de"/g' "$src"; done; else true; fi
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/htdocs ]; then install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/www; cp -pR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/htdocs/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/www/; else true; fi
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/root ]; then install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/; cp -pR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/root/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/; else true; fi
if [ -d /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src ]; then CFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " CXXFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -mips16 -minterlink-mips16  -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " LDFLAGS="-L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/lib -L/home/xiao/lede/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/lib -znow -zrelro " FPIC="-DPIC -fpic" LUCI_VERSION="git-24.182.36151-66d29de" LUCI_GITBRANCH="LuCI Master" make -C /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src/ AR="mipsel-openwrt-linux-musl-gcc-ar" AS="mipsel-openwrt-linux-musl-gcc -c -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -fmacro-prefix-map=/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base=luci-base -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro" LD=mipsel-openwrt-linux-musl-ld NM="mipsel-openwrt-linux-musl-gcc-nm" CC="mipsel-openwrt-linux-musl-gcc" GCC="mipsel-openwrt-linux-musl-gcc" CXX="mipsel-openwrt-linux-musl-g++" RANLIB="mipsel-openwrt-linux-musl-gcc-ranlib" STRIP=mipsel-openwrt-linux-musl-strip OBJCOPY=mipsel-openwrt-linux-musl-objcopy OBJDUMP=mipsel-openwrt-linux-musl-objdump SIZE=mipsel-openwrt-linux-musl-size CROSS="mipsel-openwrt-linux-musl-" ARCH="mipsel" DESTDIR="/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install"  install; cp -fpR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/* /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/; else true; fi
make[4]: Entering directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
mkdir -p /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/template
cp parser.so /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/template/parser.so
cp version.lua /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-install/usr/lib/lua/luci/version.lua
make[4]: Leaving directory '/home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/src'
find /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base -name 'CVS' -o -name '.svn' -o -name '.#*' -o -name '*~'| xargs -r rm -rf
export CROSS="mipsel-openwrt-linux-musl-"  NO_RENAME=1 ; NM="mipsel-openwrt-linux-musl-nm" STRIP="/home/xiao/lede/staging_dir/host/bin/sstrip -z" STRIP_KMOD="/home/xiao/lede/scripts/strip-kmod.sh" PATCHELF="/home/xiao/lede/staging_dir/host/bin/patchelf" /home/xiao/lede/scripts/rstrip.sh /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base
rstrip.sh: /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/usr/lib/lua/luci/template/parser.so: shared object
(cd /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base/CONTROL; ( echo "$CONTROL"; printf "Description: "; echo "$DESCRIPTION" | sed -e 's,^[[:space:]]*, ,g'; ) > control; chmod 644 control; ( echo "#!/bin/sh"; echo "[ \"\${IPKG_NO_SCRIPT}\" = \"1\" ] && exit 0"; echo "[ -s "\${IPKG_INSTROOT}/lib/functions.sh" ] || exit 0"; echo ". \${IPKG_INSTROOT}/lib/functions.sh"; echo "default_postinst \$0 \$@"; ) > postinst; ( echo "#!/bin/sh"; echo "[ -s "\${IPKG_INSTROOT}/lib/functions.sh" ] || exit 0"; echo ". \${IPKG_INSTROOT}/lib/functions.sh"; echo "default_prerm \$0 \$@"; ) > prerm; chmod 0755 postinst prerm; echo "$V_Package_luci_base_conffiles" > conffiles;  echo "$V_Package_luci_base_postinst" > postinst-pkg; chmod 0755 postinst-pkg; )
install -d -m0755 /home/xiao/lede/bin/packages/mipsel_24kc/luci
/home/xiao/lede/staging_dir/host/bin/fakeroot /home/xiao/lede/staging_dir/host/bin/bash /home/xiao/lede/scripts/ipkg-build -m "" /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base /home/xiao/lede/bin/packages/mipsel_24kc/luci
Packaged contents of /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-mipsel_24kc/luci-base into /home/xiao/lede/bin/packages/mipsel_24kc/luci/luci-base_git-24.182.36151-66d29de-1_mipsel_24kc.ipk
removed '/home/xiao/lede/bin/packages/mipsel_24kc/luci/luci-i18n-base-zh-cn_git-24.182.36151-66d29de-1_all.ipk'
mkdir -p /home/xiao/lede/bin/targets/ramips/mt7621/packages /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn/CONTROL /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/pkginfo
install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn/etc/uci-defaults
echo "uci set luci.languages.zh_cn='简体中文 (Simplified Chinese)'; uci commit luci" > /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn/etc/uci-defaults/luci-i18n-base-zh-cn
install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn/usr/lib/lua/luci/i18n
po2lmo /home/xiao/lede/feeds/luci/modules/luci-base/po/zh-cn/base.po /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn/usr/lib/lua/luci/i18n/base.zh-cn.lmo;
find /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn -name 'CVS' -o -name '.svn' -o -name '.#*' -o -name '*~'| xargs -r rm -rf
export CROSS="mipsel-openwrt-linux-musl-"  NO_RENAME=1 ; NM="mipsel-openwrt-linux-musl-nm" STRIP="/home/xiao/lede/staging_dir/host/bin/sstrip -z" STRIP_KMOD="/home/xiao/lede/scripts/strip-kmod.sh" PATCHELF="/home/xiao/lede/staging_dir/host/bin/patchelf" /home/xiao/lede/scripts/rstrip.sh /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn
(cd /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn/CONTROL; ( echo "$CONTROL"; printf "Description: "; echo "$DESCRIPTION" | sed -e 's,^[[:space:]]*, ,g'; ) > control; chmod 644 control; ( echo "#!/bin/sh"; echo "[ \"\${IPKG_NO_SCRIPT}\" = \"1\" ] && exit 0"; echo "[ -s "\${IPKG_INSTROOT}/lib/functions.sh" ] || exit 0"; echo ". \${IPKG_INSTROOT}/lib/functions.sh"; echo "default_postinst \$0 \$@"; ) > postinst; ( echo "#!/bin/sh"; echo "[ -s "\${IPKG_INSTROOT}/lib/functions.sh" ] || exit 0"; echo ". \${IPKG_INSTROOT}/lib/functions.sh"; echo "default_prerm \$0 \$@"; ) > prerm; chmod 0755 postinst prerm; echo "$V_Package_luci_i18n_base_zh_cn_postinst" > postinst-pkg; chmod 0755 postinst-pkg; )
install -d -m0755 /home/xiao/lede/bin/packages/mipsel_24kc/luci
/home/xiao/lede/staging_dir/host/bin/fakeroot /home/xiao/lede/staging_dir/host/bin/bash /home/xiao/lede/scripts/ipkg-build -m "" /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn /home/xiao/lede/bin/packages/mipsel_24kc/luci
Packaged contents of /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/ipkg-all/luci-i18n-base-zh-cn into /home/xiao/lede/bin/packages/mipsel_24kc/luci/luci-i18n-base-zh-cn_git-24.182.36151-66d29de-1_all.ipk
echo "luci-base" >> /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/pkginfo/luci-base.default.install
rm -rf /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn.installed /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn
mkdir -p /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn
install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn/etc/uci-defaults
echo "uci set luci.languages.zh_cn='简体中文 (Simplified Chinese)'; uci commit luci" > /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn/etc/uci-defaults/luci-i18n-base-zh-cn
install -d -m0755 /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn/usr/lib/lua/luci/i18n
po2lmo /home/xiao/lede/feeds/luci/modules/luci-base/po/zh-cn/base.po /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn/usr/lib/lua/luci/i18n/base.zh-cn.lmo;
touch /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn.installed
mkdir -p /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/root-ramips/stamp
SHELL= flock /home/xiao/lede/tmp/.root-copy.flock -c 'cp -fpR /home/xiao/lede/build_dir/target-mipsel_24kc_musl/luci-base/.pkgdir/luci-i18n-base-zh-cn/. /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/root-ramips/'
touch /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/root-ramips/stamp/.luci-i18n-base-zh-cn_installed
echo "luci-i18n-base-zh-cn" >> /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/pkginfo/luci-base.default.install
make[3]: Leaving directory '/home/xiao/lede/feeds/luci/modules/luci-base'
time: package/feeds/luci/luci-base/compile#2.25#5.76#7.72
make[2]: Leaving directory '/home/xiao/lede'
make[1]: *** [package/Makefile:110: /home/xiao/lede/staging_dir/target-mipsel_24kc_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/xiao/lede'
make: *** [/home/xiao/lede/include/toplevel.mk:231：world] 错误 2

**Comment by coolsnowwolf** [comment]
好了
